css/interactive-guides/circuit-breaker/checkBalance.css

```diff
@@ -22,21 +22,22 @@ html, body {
   display: flex;
   flex-wrap: nowrap;
   flex-direction: column;
-  background:rgba(231,29,50,0.07);
+  background-color: unset;
   justify-content: center;
   margin: 24px 24px 0; 
   flex-grow: 1;
   min-width: 0;
-  min-height: 0;
+  min-height: 170px;
   overflow: hidden;
 }
 
 .flexWarningContainer > div {
   text-align: center;
   line-height: 24px;
-  font-size: 14px;
+  font-weight: 400;
+  font-size: 16px;
```

Comment:
16px is too high.  NONE of the text in the guide is over 14px.   It stands out too much and should be left at 14px.

Reply:
I will add it to the list.